.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
 <!-- 
 Thank your for making Kong better! #kongstrong
 
+After creating a pull request, we will automatically generate a preview version of the docs site with your changes. You should see a comment with a link on your PR several minutes after it is created. Please review the generated preview for broken links, formatting issues, etc. if you have not already done so using a local preview.
```

Comment:
This helpful new content is in an HTML comment - can you add a screen shot to this PR showing how/where this content will be displayed to end-users?

Reply:
Similar to how the existing comment text is displayed, e.g. https://i.imgur.com/xqKyizo.png

It shouldn't go into the PR content after a contributor adds it.